src/axom/slic/internal/stacktrace.cpp

```diff
@@ -13,7 +13,10 @@
   #include <DbgHelp.h>
 #else
   #include <execinfo.h> // for backtrace()
-  #include <cxxabi.h>   // for abi::__cxa_demangle
+  #include <ciso646>
+  #if !defined(_LIBCPP_VERSION)

```

Comment:
Thanks @brunner6.

Looks like there is an issue with our mac CI test.
```
Start  34: slic_logging

34: Test command: /Users/runner/runners/2.165.1/work/1/s/build/examples/slic_logging_ex
34: Test timeout computed to be: 1500
34: slic_logging_ex(3313,0x7fffb6314380) malloc: *** error for object 0x7ff8bf402ce0: pointer being freed was not allocated
34: *** set a breakpoint in malloc_error_break to debug
34: Mon Mar  9 17:27:44 2020
```
See: https://dev.azure.com/axom/axom/_build/results?buildId=993&view=logs&j=8dab7c2a-824f-53c4-ff36-17cdfa82e5d3&t=d13b7ead-2e4c-51d6-8176-f474971cfc15&l=6790

I'm guessing it has to do with the ``free(realName)`` in line 147 when ``status==false``